src/matcha_ml/state/remote_state_manager.py

```diff
@@ -141,6 +141,19 @@ def _resource_group_exists(self) -> bool:
         """
         return self.azure_storage.resource_group_exists
 
+    def get_hash_remote_state(self, remote_path: str) -> str:
```

Comment:
Nit: should we harmonize the function names, either get_remote/locale_state_hash or get_hash_remote/local_state?

Reply:
Thanks, added `get_hash_remote_state` and `get_hash_local_state` for consistency.